common/src/main/java/com/wynntils/wc/utils/scoreboard/ScoreboardManager.java

```diff
@@ -195,6 +196,15 @@ private static void handleScoreboardReconstruction() {
 
             scoreboard.setDisplayObjective(1, objective);
 
+            // Set player team display objective
+            PlayerTeam playerTeam = scoreboard.getPlayersTeam(McUtils.player().getScoreboardName());
```

Comment:
What does this do?

Reply:
This fixes flickering because of a vanilla thing.

Reply:
Aha. Maybe leave a comment saying something like that?